Userland/Libraries/LibWeb/Painting/CheckBoxPaintable.cpp

```diff
@@ -33,4 +90,72 @@ Layout::CheckBox& CheckBoxPaintable::layout_box()
     return static_cast<Layout::CheckBox&>(layout_node());
 }
 
+void CheckBoxPaintable::paint(PaintContext& context, PaintPhase phase) const
+{
+    if (!is_visible())
+        return;
+
+    PaintableBox::paint(context, phase);
+
+    if (phase == PaintPhase::Foreground) {
+        auto const& checkbox = static_cast<HTML::HTMLInputElement const&>(layout_box().dom_node());
+        bool enabled = layout_box().dom_node().enabled();
+        Gfx::AntiAliasingPainter painter { context.painter() };
+        auto checkbox_rect = context.enclosing_device_rect(absolute_rect()).to_type<int>();
+        auto checkbox_radius = checkbox_rect.width() / 5;
+
+        auto& palette = context.palette();
+
+        auto shade = [&](Color color, float amount = 0.3f) {
```

Comment:
It feels arbitrary to have a default `amount = 0.3f` when you specify the amount almost everywhere else. Why not always set the required `amount` in the calling code?

Reply:
No real reason (other than `0.3f` felt like a nice default increment between shades, but I'll remove it).